script/check-publish-package.js

```diff
@@ -33,7 +33,6 @@ const FILES_CHECKLIST = [
   'es/HyperFormula.js',
   'typings/index.d.ts',
   'typings/HyperFormula.d.ts',
-  'languages/all.js',
```

Comment:
Maybe we should check `/dist/languages/all.js` then? Might be a good idea to check that language budnles were generated